contrib/camel/config/conditions_patch.yaml

```diff
@@ -0,0 +1,17 @@
+# this fixes the auto-detected object, we use a string in the stored object but
```

Comment:
We have removed kustomize from the project, fyi.

Reply:
Going to refactor!